README.md

```diff
@@ -4,8 +4,8 @@ CUDA Python is the home for accessing NVIDIA’s CUDA platform from Python. It c
 
 * [cuda.core](https://nvidia.github.io/cuda-python/cuda-core/latest): Pythonic access to CUDA Runtime and other core functionalities
 * [cuda.bindings](https://nvidia.github.io/cuda-python/cuda-bindings/latest): Low-level Python bindings to CUDA C APIs
-* [cuda.cooperative](https://nvidia.github.io/cccl/cuda_cooperative/): Pythonic exposure of CUB cooperative algorithms
-* [cuda.parallel](https://nvidia.github.io/cccl/cuda_parallel/): Pythonic exposure of Thrust parallel algorithms
+* [cuda.cooperative](https://nvidia.github.io/cccl/cuda_cooperative/): Exposure of CUB cooperative algorithms for use in Python kernels
+* [cuda.parallel](https://nvidia.github.io/cccl/cuda_parallel/): Pythonic exposure of Thrust parallel algorithms launched from the host
```

Comment:
Question, based on this observation:
```
$ cd cccl/c/parallel
$ git grep -i '#include.*cub' | wc -l
44
$ git grep -i '#include.*thrust' | wc -l
0
```

I.e., currently thrust headers aren't used at all in cuda.parallel.

Could it be better to write
```
Pythonic exposure of CCCL parallel algorithms launched from the host
```
?

Reply:
How about being a bit more explicit here?

* [cuda.parallel](https://nvidia.github.io/cccl/cuda_parallel/): A Python package offering highly efficient and customizable implementations of parallel algorithms like sort, reduce, scan, transform, etc.,

* [cuda.cooperative](https://nvidia.github.io/cccl/cuda_cooperative/): A Python package providing [CUB's](https://nvidia.github.io/cccl/cub/index.html) reusable block-wide and warp-wide primitives for use within Numba CUDA kernels.

I like putting `cuda.parallel` first as it's probably a more recognizable feature set compared to `cuda.cooperative`.

Reply:
I like Ashwin's version. Maybe replace "offering" with "for easy access to"?

Reply:
> currently thrust headers aren't used at all in cuda.parallel

An accurate reflection of implementation detail in the package name was not the intent. On the GPU side, everything is actually CUB and Thrust is a sort of C++ friendly wrappers on top, so the intent was to capture the spirit that we get to answer the question "I need a sort that can be launched from host like `thrust::sort`" with the answer "no problem, use `cuda.parallel.sort`."